src/mongoose/json-transform.js

```diff
@@ -78,25 +78,38 @@ function translateDoc(doc, ret, options) {
   return i18n(ret, options.langs, options.defaultLanguage, options.includeTranslations);
 }
 
+
+function _generateUrl(parts) {
+  var url = parts.join('/');
+
+  // this is a bit grim but there doesn't seem to be a JS
+  // library to do this.
+  var reDoubleSlash = /([^:])\/\//g;
+  url = url.replace(reDoubleSlash, "$1/");
```

Comment:
I think you can simplify this by using [`path.join`](http://nodejs.org/api/path.html#path_path_join_path1_path2) which will handle double slashes correctly.